wgpu-types/src/features.rs

```diff
@@ -1509,6 +1509,19 @@ impl Features {
         ]))
     }
 
+    /// Mask of all features which are experimental.
+    #[must_use]
+    pub const fn all_experimental_mask() -> Self {
```

Comment:
Add something to ensure that this is kept up to date. For example, a test roughly like this (not trying to make the types work out):

```rust
#[test]
fn all_experimental_features_are_listed() {
    for (name, _) in FeaturesWGPU::all().difference(all_experimental_mask()) {
        assert!(!name.contains("EXPERIMENTAL"), "{name}");
    }
}
```